spec/features/applications/new_spec.rb

```diff
@@ -43,7 +43,7 @@
         fill_in "Status", with: "In Progress"
         click_button "Submit"
 
-        expect(current_path).to eq("/applications/#{application.id}")
+        expect(current_path).to eq("/applications/#{@app2.id + 1}")
 
```

Comment:
Why do we need to increment in the id?

Reply:
`application` is not a stored variable in this scope, but we do know that AR increments new object ids as it adds new records.
always better to test the expected results than to test the NOT version of something not supposed to be there (the previous test was "expect page path to not be same" which is a weaker test than the positive form)